go/cmd/bpf/bpf.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"os"
+
+	usercorn "github.com/lunixbochs/usercorn/go"
```

Comment:
Doesn't this already import as usercorn? Are you just being explicit?

Reply:
`vim-go` decided to format it this way when I ran `:GoImports`. I can fix it.